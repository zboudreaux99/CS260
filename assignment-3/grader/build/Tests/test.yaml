test:
  timelimit: 10
dueDate: 05/23/2023
latePenalties: [0, 10, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["readingList.h", "readingList.cpp"]
  instructorFiles:
    exclude: ["readingList.h", "readingList.cpp"]
