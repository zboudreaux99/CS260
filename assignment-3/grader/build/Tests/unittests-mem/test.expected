1..9
ok 1 - testRLAddDiscrete
ok 2 - testRLAddDuplicates
ok 3 - testRLAssign
ok 4 - testRLConstructor
ok 5 - testRLCopy
ok 6 - testRLDefaultConstructor
ok 7 - testRLInput
ok 8 - testRLRemove
ok 9 - zzMemoryCheck
# UnitTest: passed 9 out of 9 tests, for a success rate of 100.0%

--- end of output ---
--- std error ---
Leaked memory: 0 blocks
